cmd/utils.go

```diff
@@ -131,7 +135,11 @@ func getVolumeArgs() []string {
 		homeDir = homeDirOverride
 		homeDirOverridden = true
 	}
-	projectDir := getProjectDir()
+	projectDir, perr := getProjectDir()
+	if perr != nil {
+		Error.log("The current directory is not a valid appsody project. Run appsody init <stack> to create one: ", perr)
```

Comment:
Same comment as above, duplicate message?

Reply:
This has been resolved now.